src/config/__mocks__/package.js

```diff
@@ -0,0 +1,93 @@
+import collector from './../../collector';
```

Comment:
I'm not sure about mocking almost all of the functionality of this piece of the puzzle. I would think the only thing to be mocked might be the "source" of where the plugin pulls its data from (package.json).

Reply:
Would love to know how to do that. I attempted to import the `PackageConfig` into my mock and it resulted in a jest recursion error. Appears jest's support for mocking classes is hit-or-miss.

Reply:
I see. One strategy may be to pull out the function that loads the file (package.json), and export it using a named export. Then you can just mock that piece I believe?

Reply:
An example of this is in the trace plugin. I pulled the function into it's own file but that's not entirely necessary if you don't want to go that route.
https://github.com/iopipe/iopipe-plugin-trace/blob/master/src/__mocks__/addToReport.js